README.md

```diff
@@ -95,6 +95,9 @@ Environment variables used for external services include:
 - MAGIC_API_KEY
 - MAGIC_SUPPORTED_BASES
 - MAGIC_DEFAULT_CHAIN
+- DISCORD_CLIENT_ID: for Discord OAuth login
+- DISCORD_CLIENT_SECRET: for Discord OAuth login
+- DISCORD_OAUTH_SCOPES: scopes (usually just 'identify')
```

Comment:
This variable unused?

Reply:
Not sure what you mean they are all being used.

Reply:
Can you explain what scopes does?

Reply:
It's just the permissions that a user agrees to give us (in terms of access to their account). So for example, "identify" allows us to get their username/profile info.